packages/block-editor/src/components/block-list/use-block-props/use-focus-first-element.js

```diff
@@ -52,6 +52,16 @@ function useInitialPosition( clientId ) {
 	);
 }
 
+function isFormElement( element ) {
```

Comment:
I borrowed this function from writing-flow. Should probably throw this somewhere else in the future.